src/translations/strings.yaml

```diff
@@ -575,14 +578,18 @@ onboarding:
   privacySlideHeader: Get more privacy
   privacySlideBody: Use these features for more protection. They may cause issues on some sites, so you can turn them off anytime in settings.
   devicesSlideHeader: Install Mozilla VPN on up to 5 devices
-  devicesSlideBody: Scan this QR code using a tablet or mobile device’s camera to install the VPN on another device.
+  devicesSlideBody2: To install the VPN on another tablet or mobile device, scan the QR code.
+  devicesSlideBodyMobile: To install the VPN on another tablet or mobile device, scan the QR code. To install on a desktop computer, visit the Download page on the Mozilla VPN website.
```

Comment:
Nitpicking, but there is no Mozilla VPN website, there's a mozilla.org website with VPN pages.

Reply:
Ok, I would leave this one, because you can go to Google and search "Mozilla VPN", and you do get what looks like a dedicated site as the first hit. I know it's technically pages within the Mozilla site, but being more accurate here doesn't really seem like it'll add any value, imo.